courses/serializers.py

```diff
@@ -45,15 +45,26 @@ def _get_thumbnail_url(page):
 class BaseCourseSerializer(serializers.ModelSerializer):
     """Basic course model serializer"""
 
+    type = serializers.SerializerMethodField(read_only=True)
+
     def to_representation(self, instance):
         data = super().to_representation(instance)
         if not self.context.get("include_page_fields") or not hasattr(instance, "page"):
             return data
         return {**data, **CoursePageSerializer(instance=instance.page).data}
 
+    @staticmethod
+    def get_type(obj):
+        return "course"
```

Comment:
I think we can use the `CONTENT_TYPE_MODEL_PROGRAM` and `CONTENT_TYPE_MODEL_COURSE` constants.